src/routes/controllers/articles.controller.js

```diff
@@ -74,7 +75,7 @@ export async function createArticle(req, res) {
       createdArticle,
     );
   } catch (error) {
-    return errorResponse(res, 500, error.message);
+    return serverError(res, 500, message);
```

Comment:
Thanks for taking one for the team. I noticed you passed a message variable everytime you called this helper function but I cannot see where it comes.